website_v2/gcloud-powershell-website-v2/www/static/css/maia.css

```diff
@@ -1 +1 @@
maia-plusone,#maia-header .maia-util .maia-search{float:left;margin:0 0 0 15px}[dir=rtl] #maia-header .maia-util .maia-plusone,[dir=rtl] #maia-header .maia-util .maia-search{float:right;margin:0 15px 0 0}#maia-header .maia-util>:first-child{margin:0}#maia-header .maia-util .maia-plusone{display:block;line-height:normal;margin-top:10px !important}#maia-header .maia-util .maia-search{margin-top:8px !important}#maia-header .maia-util a+.maia-button{margin:5px 0 5px 10px}[dir=rtl] #maia-header .maia-util a+.maia-button{margin:5px 10px 5px 0}*+html #maia-header .maia-util a+.maia-button{vertical-align:middle}#maia-header .maia-plusone{display:inline;line-height:50px;margin:0 0 0 10px}[dir=rtl] #maia-header .maia-plusone{margin:0 10px 0 0}#maia-header .maia-search{float:right;margin:0}[dir=rtl] #maia-header .maia-search{float:left}.maia-breadcrumb{font-size:11px;line-height:1.82;margin:0 0 20px;min-height:28px;overflow:hidden}.maia-breadcrumb ol{margin:0 auto;max-width:978px;padding:12px 0}.maia-breadcrumb ol:after{clear:both;content:" ";display:block;height:0;visibility:hidden}.maia-breadcrumb li{float:left;font-weight:bold;list-style:none}[dir=rtl] .maia-breadcrumb li{float:right}.maia-breadcrumb li a{font-weight:normal;margin:0 5px 0 0;padding:1px 0 0}[dir=rtl] .maia-breadcrumb li a{margin:0 0 0 5px}.maia-breadcrumb li+li:before{content:"â€º";padding:0 6px 0 3px}[dir=rtl] .maia-breadcrumb li+li:before{padding:0 3px 0 6px}* html .maia-breadcrumb li{margin:0 13px 0 0}*+html .maia-breadcrumb li{margin:0 13px 0 0}*+[dir=rtl] .maia-breadcrumb li{margin:0 0 0 13px}.maia-button{background-color:#4173c9;background-image:-moz-linear-gradient(top,#5e8ee4,#4173c9);background-image:-webkit-linear-gradient(top,#5e8ee4,#4173c9);background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#5e8ee4),color-stop(1,#4173c9));background-image:linear-gradient(top,#5e8ee4,#4173c9);border:0;-moz-border-radius:2px;-webkit-border-radius:2px;border-radius:2px;color:#fff;cursor:pointer;display:inline-block;font-family:arial,sans-serif;font-size:13px;font-weight:bold;line-height:1.54;padding:7px 12px;text-align:center;text-shadow:0 -1px 0 rgba(0,0,0,.08)}a.maia-button{color:#fff !important;text-decoration:none}button.maia-button{margin:0;overflow:visible;white-space:normal}button.maia-button::-moz-focus-inner{border:0;padding:0}.maia-button:hover,.maia-button:focus{background-color:#396bbc;background-image:-moz-linear-gradient(top,#5587d7,#396bbc);background-image:-webkit-linear-gradient(top,#5587d7,#396bbc);background-image:-webkit-gradient(linear,left top,left bottom,color-stop(0,#5587d7),color-stop(1,#396bbc));background-image:linear-gradient(top,#5587d7,#396bbc)}.maia-button:active{-moz-box-shadow:inset 0 2px 4px rgba(0,0,0,.24);-webkit-box-shadow:inset 0 2px 4px rgba(0,0,0,.24);box-shadow:inset 0 2px 4px rgba(0,0,0,.24)}.maia-button[disabled],.maia-button.maia-button-disabled{background-color:#a4b9e3;background-image:-moz-linear-gradient(top,#b0c7f0,#a4b9e3);
```

Comment:
(before the line was
 #maia-footer-global li+li:before{content:"Â·";padding:0 3px}* html
now it's
#maia-footer-global li+li:before{content:"·";padding:0 3px}* html
)